src/components/Input/Input.tsx

```diff
@@ -3,7 +3,17 @@ import cx from "classnames";
 
 import "./input.css";
 
-const Input = ({ className, fluid, ...props }) => {
+type Props = React.HTMLAttributes<HTMLInputElement> &
+  React.HTMLProps<HTMLInputElement> & {
+    className?: string;
```

Comment:
It's possible that `className` is already included with `React.HTMLProps<HTMLInputElement`?